internal/server/users_test.go

```diff
@@ -422,7 +422,8 @@ func TestAPI_CreateUser(t *testing.T) {
 		body := jsonBody(t, tc.body)
 		req, err := http.NewRequest(http.MethodPost, "/api/users", body)
 		assert.NilError(t, err)
-		req.Header.Add("Authorization", "Bearer "+adminAccessKey(srv))
+		req.Header.Set("Authorization", "Bearer "+adminAccessKey(srv))
+		req.Header.Set("Infra-Version", "0.13.6")
```

Comment:
Should we automatically set this to the current version to reveal breaking changes? I think that is possible.